src/xo-app/logs/index.js

```diff
@@ -85,23 +85,28 @@ const LOG_COLUMNS = [
   },
   {
     name: _('jobId'),
-    itemRenderer: log => log.jobId
+    itemRenderer: log => log.jobId,
+    sortCriteria: log => log.jobId
   },
   {
     name: _('job'),
-    itemRenderer: log => jobKeyToLabel[log.key]
+    itemRenderer: log => jobKeyToLabel[log.key],
+    sortCriteria: log => jobKeyToLabel[log.key]
```

Comment:
Will this behave correctly as `jobKeyToLabel[log.key]` returns a React component?